src/helpers/FormioLoader/FormioLoader.js

```diff
@@ -0,0 +1,171 @@
+import defaultCreateFormOptions from "../../config/createForm.options";
+import defaultCreateFormController from "../../config/createForm.controller";
+
+// polyfill plugin function to fix the namespace option doesn't pass to Formio.makeRequest
+const NamespacePlugin = {
```

Comment:
why is this showing up as a new file?

Reply:
i've rename src/helpers/FormioLoader/index.js to src/helpers/FormioLoader/FormioLoader.js
seems github didn't detect it as rename action...